src/App.js

```diff
@@ -45,10 +44,15 @@ class App extends React.Component {
       this.setState({
         cityData: citySearch.data[0],
         showMap: true,
+        imgUrl: `https://maps.locationiq.com/v3/staticmap?key=${process.env.REACT_APP_LOCATIONIQ_ACCESS_TOKEN}&center=${this.state.cityData.lat},${this.state.cityData.lon}&zoom=10`,
         errorMessage: ''
 
       })
     } catch (error) {
+      let url = `http://localhost:3002/throw-an-error`
+      let errorMessage = await axios.get(url);
+      console.log(errorMessage);
+
```

Comment:
- I am curious to know what this is doing here 👍  It is making a call to the error message endpoint but it looks like we are handling errors below as well on line 56-58.